asset/js/mapping-collecting-form.js

```diff
@@ -13,8 +13,8 @@ $(document).ready(function() {
         }).addTo(map);
         map.setView([20, 0], 2);
 
-        map.addControl(new L.Control.GeoSearch({
-            provider: new L.GeoSearch.Provider.OpenStreetMap(),
+        map.addControl(new window.GeoSearch.GeoSearchControl({
+            provider: new window.GeoSearch.OpenStreetMapProvider,
```

Comment:
What's the rationale for switching from `L.Control.GeoSearch()` to `window.GeoSearch`?

Reply:
Geosearch is now an independant plugin, that can work without Leaflet, so it is attached by default to the window. Some more works in the javascript is needed to integrate it in the Leaflet, but it is useless.